scripts/test-proxy-startup-with-env-vars.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -e  # Exit on any error
+
+##### This script is meant to be run in ci #####
+# It tests that the env vars defined in .env.example.holesky are correct.
+# It starts the eigenda-proxy with those env vars, waits 5 seconds, and then kills the proxy.
+# If any deprecated flags are still being used in .env.example.holesky, the script will fail.
```

Comment:
shouldn't we also test set/get logic as well? or at a minimum ping the health endpoint to at least ensure the server instance successfully finished bootstrapping?

Reply:
[9204b0d](https://github.com/Layr-Labs/eigenda-proxy/pull/286/commits/9204b0d5d7dc2ae8e62d8cbb4b73c083423ed3c2)